File: 0389-find-the-difference/0389-find-the-difference.php
class Solution {

    /**
     * @param String $s
     * @param String $t
     * @return String
     */
    function findTheDifference($s, $t) {
        $sArr = str_split($s);
        $tArr = str_split($t);
        
        foreach($tArr as $tItem){
            if(!in_array($tItem, $sArr)){
                return $tItem;
            }
            
            unset($sArr[array_search($tItem, $sArr)]);
        }
    }
}

File: 0389-find-the-difference/0389-find-the-difference.go
func findTheDifference(s string, t string) byte {
    var c = t[len(s)];
    
    for  i:= 0; i < len(s); i++ {
		c ^= s[i];
		c ^= t[i];
	}   
    
	return c;
}